llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -10,20 +10,36 @@
 // RISC-V processors supported.
 //===----------------------------------------------------------------------===//
 
+class RISCVTuneInfo {
```

Comment:
Should these fields be in RISCVProcessorModel if we're always going to have RISCVProcessorModel inherit this class?